tutorials/repository_sync_dbfs.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Instead of hard-code the JSON as raw string, do you mind add a few lines to save PyDeequ metrics results to the metrics file location above? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good idea - let me add it</p>

Reply:
<p>Updated</p>